pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+asyncio_mode = auto
+asyncio_default_fixture_loop_scope = "function"
```

Comment:
Without this, I get:

![Screenshot 2025-02-27 at 1 18 34 pm](https://github.com/user-attachments/assets/63805b2b-58f4-4adc-a7fc-f76df8274b0e)